extension/content_script.js

```diff
@@ -51,39 +51,25 @@ function getContainer(containerId) {
 
 // Each readable item
 // Assumes that readableDomId is already populated
+// TODO: Refactor this with setTracker().
 function setupClickListener() {
 	for (let i = 0; i < readableDomIds.length; i++) {
 		getContainer(i).click(function () {
 			containerId = i;
-			let container = getContainer(containerId);
+			container = getContainer(containerId);
 			start = 0;
 			let txt = container.text();
 			end = txt.indexOf(". ", start);
 			if (end < 0) {
 				end = txt.length;
 			}
 			highlight(container, start, end);
-			containerInit = 1; // Container has now been initialized
-			if (!startReading) {startReading = true}; // We have now started reading
 		});
 	}
 }
 
 // Move tracker
 function move(type) { // Note: I have combined the "moveUp" and "moveDown" functions here
-	if (!startReading) { // If user has not started reading yet
```

Comment:
The same initialization logic exists in setTracker. Remove because redundant.

Reply:
LGTM